types/helpers/helpers.easing.d.ts

```diff
@@ -0,0 +1,35 @@
+export type EasingFunctionSignature = (t: number) => number;
+
+export const easing: {
```

Comment:
This is guaranteed to be the same listing as type `EasingFunction`, right? You can simplify this declaration as:

```typescript
export const easing: Record<EasingFunction, EasingFunctionSignature>;
```